Network/src/Game/game_command_ter.c

```diff
@@ -10,16 +10,16 @@
 #include "Game/game_functions.h"
 #include "Server/server.h"
 
-void set_object_down(client_t *client, server_t *server, char *args)
+void set_object_down(client_t *client, server_t *server, char **args)
 {
-    bool is_ok = drop_object(client, server, args);
+    bool is_ok = drop_object(client, server, args[0]);
 
     send(client->socket, is_ok ? "ok\n" : "ko\n", 3, 0);
```

Comment:
maybe create a function for this

```suggestion
    sendStatus(client, is_ok);
}

void sendStatus(client_t *client, bool status)
{
    send(client->socket, is_ok ? "ok\n" : "ko\n", 3, 0);
}
```
to avoid rewriting the same line over and over

Reply:
nah im good